### PR TITLE
Add v13.0.0 for AWS

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

**Comment by calvix** [comment]
manual cncf test with sonobuy works but this not, I have no idea what is wrong

**Comment by MarcelMue** [comment]
> manual cncf test with sonobuy works but this not, I have no idea what is wrong

I will take a look tomorrow morning first thing

**Comment by calvix** [comment]
of course it works now

File: aws/v13.0.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    version: 0.1.0
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 3.3.1
+  - name: containerlinux
+    version: 2512.5.0
+  - name: etcd
+    version: 3.4.13
+  - name: kubernetes
+    version: 1.18.10
```
Comment (tfussell) [inline aws/v13.0.0/release.yaml:65]: You're probably still working on this, but I'm going with Kubernetes 1.18.12 in KVM 13.0.0 if you want to keep them in sync.

File: aws/v13.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+#### Dependencies
+#### Added
+_Nothing has changed._
+#### Changed
+_Nothing has changed._
+#### Removed
+_Nothing has changed._
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:61]:
```suggestion
```

File: aws/v13.0.0/README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+-  Allow annotations from current app CR to remain.
+
+### app-operator [2.7.0](https://github.com/giantswarm/app-operator/releases/tag/v2.7.0)
+
+#### Added
+- Secure the webhook with token value from control plane catalog.
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:89]:
```suggestion
```

File: aws/v13.0.0/README.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+#### Added
+- Validate the cache in helmclient to avoid state requests when pulling tarballs.
+- Call status webhook with token values.
+#### Fixed
+- Update apiextensions to v3 and replace CAPI with Giant Swarm fork.
+- Fix comparison of last deployed and revision optional fields in status resource.
+- Set memory limit and reduce requests.
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:101]:
```suggestion
```

File: aws/v13.0.0/README.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+### kube-state-metrics [1.3.0](https://github.com/giantswarm/kube-state-metrics-app/releases/tag/v1.3.0)
+
+#### Changed
+- Change the Kubernetes Deployment name to include the app version.
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:108]:
```suggestion
```

File: aws/v13.0.0/README.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+### cert-exporter [1.3.0](https://github.com/giantswarm/cert-exporter/releases/tag/v1.3.0)
+
+#### Added
+- Add Network Policy.
+#### Changed
+- Remove `hostNetwork` and `hostPID` capabilities.
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:117]:
```suggestion
```

File: aws/v13.0.0/README.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+### net-exporter [1.9.2](https://github.com/giantswarm/net-exporter/releases/tag/v1.9.2)
+
+#### Changed
+- Updated backward incompatible Kubernetes dependencies to v1.18.5.
+#### Fixed
+- Fixed indentation problem with the daemonset template.
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:126]:
```suggestion
```

File: aws/v13.0.0/README.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+### node-exporter [1.7.0](https://github.com/giantswarm/node-exporter-app/releases/tag/v1.7.0)
+
+#### Changed
+- Change the Kubernetes Daemonset name to include the app version.
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:133]:
```suggestion
```

File: aws/v13.0.0/README.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+### external-dns [1.5.0](https://github.com/giantswarm/external-dns-app/releases/tag/v1.5.0)
+
+#### Changed
+- Upgrade upstream external-dns from v0.7.3 to [v0.7.4](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.7.4).
+
+
+
```
Comment (njuettner) [inline aws/v13.0.0/README.md:140]:
```suggestion
```

**Review by njuettner — APPROVED** [review]
only suggesting to remove a line to keep it consistent

**Review by xh3b4sd — APPROVED** [review]

**Review by calvix — APPROVED** [review]

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**